google-cloud-pubsub/support/doctest_helper.rb

```diff
@@ -279,6 +279,16 @@ def mock_pubsub
     end
   end
 
+  ##
+  # RetryPolicy
+
+  doctest.before "Google::Cloud::PubSub::RetryPolicy" do
+    mock_pubsub do |mock_publisher, mock_subscriber|
+      mock_subscriber.expect :get_subscription, subscription_resp("my-topic-sub", dead_letter_topic: "my-dead-letter-topic", max_delivery_attempts: 10), ["projects/my-project/subscriptions/my-topic-sub", Hash]
```

Comment:
Is there a reason why dead letter policy is mentioned here?

Reply:
Thanks for catching this. Removed.